daffodil-io/src/main/scala/org/apache/daffodil/io/InputSource.scala

```diff
@@ -145,16 +173,21 @@ abstract class InputSource {
  *
  * @param inputStream the java.io.Inputstream to read data from
  * @param bucketSize the size of each individual bucket
- * @param maxNumberOfBuckets the maximum number of buckets that can be created before beginning to remove old ones
+ * @param maxCacheSizeInBytes the maximum number of buckets that can be created before beginning to remove old ones
```

Comment:
Description isn't right or doesn't match name of parameter. This is the total size of the buckets, not the number of buckets.